types/node/index.d.ts

```diff
@@ -6367,20 +6362,26 @@ declare module "http2" {
     }
 
     export interface Http2Session extends events.EventEmitter {
-        destroy(): void;
+        readonly alpnProtocol?: string;
+        close(callback?: () => void): void;
+        readonly closed: boolean;
+        destroy(error?: Error, code?: number): void;
         readonly destroyed: boolean;
+        readonly encrypted?: boolean;
+        goaway(code?: number, lastStreamID?: number, opaqueData?: Buffer | DataView /*| TypedArray*/): void;
```

Comment:
Should we add a `TypedArray` alias in the meantime until typescript offers a built-in solution in one of their lib files?

> `type TypedArray = Int8Array | Uint8Array | Int16Array | Uint16Array | Int32Array | Uint32Array | Uint8ClampedArray | Float32Array | Float64Array;`
>
> -- [<cite>Microsoft/TypeScript#15402</cite>](https://github.com/Microsoft/TypeScript/issues/15402#issuecomment-297544403)

Reply:
Is that going to cause re-definition issues when the built-in `TypedArray` becomes available? If so maybe it's better to either not include, or have a type with a different name. Since other places use `TypedArray` as well, it shouldn't be difficult to substitute all in a separate PR.

Reply:
As long as it's not defined in the global scope, you should be okay.